### DISCUSSION TITLE
Consider adding a blank seaorm Example without any other framework

### DISCUSSION THREAD

**Discussion by aadi58002** [SeaQL/sea-orm#1271] [Q&A] [answered]
Would it be relevent to add a blank example for people who are unfamiliar with database migration and the structure of seaorm to get a refrence at without the cluter of another framework?

I am new to databases and today studied about what are database migration and it would have been really helpful to have a reference black project which can be used to understand the structure of a sea orm.

**Comment by billy1624** [comment, marked as answer]
Hey @aadi58002, welcome to the world of database and Rust!

Check this example https://github.com/SeaQL/sea-orm/tree/master/examples/basic to taste the pure SeaORM. And I recommend this step-by-step tutorial as well https://www.sea-ql.org/sea-orm-tutorial/.

Cheers!

**Reply by tyt2y3** [reply]
The 'basic' example is as basic as it can be.

**Reply by aadi58002** [reply]
Yes I am currently in the process of reading the tutorial and it is easier to get what is going after I read about them in the tutorial.

**Comment by aadi58002** [comment]
Thanks I will look into it.

**Reply by billy1624** [reply]
Let me know if there is anything we can add into / clarify in the documentation :)

A PR would be more than welcome hahaaa
